panel_view.py

```diff
@@ -132,15 +132,15 @@ def on_post_window_command(self, window, command_name, args):
 class JustSavedBufferController(sublime_plugin.EventListener):
     def on_post_save_async(self, view):
         bid = view.buffer_id()
-        State['awaited_views'].add(bid)
+        State['just_saved_buffers'].add(bid)
 
     def on_pre_close(self, view):
         bid = view.buffer_id()
-        State['awaited_views'].discard(bid)
+        State['just_saved_buffers'].discard(bid)
 
     def on_modified_async(self, view):
         bid = view.buffer_id()
-        State['awaited_views'].discard(bid)
+        State['just_saved_buffers'].discard(bid)
```

Comment:
So, this means, if I save, then edit, before the panel would be shown, the panel is canceled. At least, that's the behaviour I see now. Not saying that is wrong, but is that also the intention here?

Reply:
Yes, that's exactly the intention here.

Reply:
Excellent, makes sense to me. 👍🏻